src/components/dialog.js

```diff
@@ -152,11 +153,15 @@ export class Dialog {
     } else {
       this.show_();
     }
-
-    return iframe.whenReady().then(() => {
-      this.buildIframe_();
-      return this;
-    });
+    return this.doc_
+      .addToFixedLayer(iframe.getElement(), /* force transfer */ true)
```

Comment:
Hmm. Should we replace `this.doc_.getBody().appendChild(...)` above with `addToFixedLayer`. Basically that's the meaning of the `addToFixedLayer()` really - it adds the element into some sort of prioritized container.

Reply:
makes sense, will do